src/runner.rs

```diff
@@ -191,13 +194,16 @@ pub mod platform {
     use winreg::enums::*;
 
     pub fn firefox_default_path() -> Option<PathBuf> {
+        if let Some(path) = find_binary("firefox.exe") {
+            return Some(path)
+        }
         let opt_path = firefox_registry_path().unwrap_or(None);
```

Comment:
I’m ambivalent. I suggested doing this for Windows for consistency, but I think it’s rather unlikely that a Windows user has Firefox somewhere in `%path%` in the first place.